modules/tensor_mechanics/include/materials/StressUpdateBase.h

```diff
@@ -73,6 +73,12 @@ class StressUpdateBase : public Material
   /// Sets the value of the global variable _qp for inheriting classes
   void setQp(unsigned int qp);
 
+  /**
+   * If updateState is not called during a timestep, this will be.  This method allows derived
+   * classes to set internal parameters from their Old values, for instance
+   */
+  virtual void propagateQpProperties();
```

Comment:
I'd make this pure virtual:
`virtual void propagateQpProperties() = 0;`
That way, we will make sure that every material that derives from this will have thought about what the right thing to do here is.  There are a handful of models in TensorMechanics that will need to be updated to do this. 

We also have some models in BISON that would need to implement this.  Maybe the best way to handle this is to go ahead and merge your code as-is.  I can then add this to all of the BISON models, and then we can make it pure virtual after that is merged in.